### ISSUE TITLE
EIPIP Meeting 107

### ISSUE THREAD

**Issue by poojaranjan** [ethcatherders/EIPIP#346]
# Date and Time

July 17, 2024 at 14:00 UTC 

# Location

Zoom: TBA in the Discord #eip-editing channel

YouTube Recording: [EIPIP Meetings](https://www.youtube.com/playlist?list=PL4cwHXAawZxpLrRIkDlBjDUUrGgF91pQw)

# Agenda

### Edit to `Final` Proposals
- [x] https://github.com/ethereum/ERCs/pull/403
- [x] https://github.com/ethereum/ERCs/pull/342

### Update to EIP-1
_TBA_

#### Misc Issue/ requires attention?
- [x] https://github.com/ethereum/ERCs/pull/524
- [x] https://github.com/ethereum/EIPs/pull/8590
- [x] https://github.com/ethereum/EIPs/pull/8673 (Close PR?)
- [x] https://github.com/ethereum/ERCs/pull/540 (Updates to `Stagnant` proposal)
- [x] https://github.com/ethereum/EIPs/issues/8734 (spam?)

### to `Draft`
- [x] https://github.com/ethereum/EIPs/pull/8724

 
### Call for Input

Call For Input | Status | Inclination/Result | Deadline
-- | -- | -- | --
[343](https://github.com/ethcatherders/EIPIP/issues/343) | Open |  |July 28, 2024
[345](https://github.com/ethcatherders/EIPIP/issues/345) | Open |  |Aug 02, 2024
[348](https://github.com/ethcatherders/EIPIP/issues/348) | Open |  |Aug 17, 2024

## 2. Other discussions and updates from past meetings
- [x] Working Group update
- [x] Security Considerations and Security Disclosures in EIPs ref: [Comment](https://github.com/ethcatherders/EIPIP/issues/346#issuecomment-2228975727)

## 3. EIPs Insight - Monthly EIPs status reporting.

Repo | Issues | PRs | Total
-- | -- | -- | --
EIP |  7 | 62  | 69
ERC | 6  | 73  |79
RIP |  2 | 12 | 14

- [EIPsInsight.com](https://eipsinsight.com/insight/2024/7)
- [Hackmd](https://hackmd.io/@poojaranjan/EthereumImprovementProposalsInsight)

## 4. [EIP Editing Office Hour](https://hackmd.io/@poojaranjan/EIPEditingOfficeHour)
- Agenda [41](https://github.com/ethcatherders/EIPIP/issues/347) Video [40](https://youtu.be/gAtkU7YeSRU)

## 5. Review action items from earlier meetings
- [Summary meeting 106](https://github.com/ethcatherders/EIPIP/issues/340#issuecomment-2207885894)
Action items from the last meeting
- Make a _Call For Input_ to collect thoughts from other Editors for https://github.com/ethereum/EIPs/pull/8390.

## Next Meeting date & time - Aug 21, 2024 (3rd Wed of the Month) at 14:00 UTC

**Comment by Dexaran** [comment]
@poojaranjan I would like to reiterate on Security Considerations and security disclosures in EIPs. 

During the previous call 105 I was raising this general problem and highlighted the state of ERC-20 problem as well. I was asked to provide links to previous discussions and proposals, so here is my comment that stores this information https://github.com/ethcatherders/EIPIP/issues/340#issuecomment-2207486824

So, my points:

1. **Informational EIP describing ERC-20 problems**. EIP editors told me to create one. I created it. EIP editors rejected it. The problem with ERC-20 standard persists. More money is lost. I would like to revive that EIP and I think that it is quite important to **document** known issues for implementors.
2. **"Final = no change" rule resulted in a situation where an issue which is known for 7 years keeps causing financial damage to  Ethereum community.** It is reasonable to apply this rule to specification section of the EIP, but not to "security considerations" because new issues can be discovered after EIP is finalized. 
3. **My proposal: implement a new rule for EIP process: "If something caused financial damage and it is transparently verifiable through blockchain history - add it to the security considerations section".**

**Comment by poojaranjan** [comment]
## Summary
### 1. Discuss Open Issues/PRs, and other topics
**https://github.com/ethereum/ERCs/pull/403**
- Too many changes to a `Final` EIP. Should not be encouraged. 
- All editors on the call decided to close the PR. 

 **https://github.com/ethereum/ERCs/pull/342**
- One of the author found it okay at high level but needs to be verified. 
- @lightclient added a [comment](https://github.com/ethereum/ERCs/pull/342#issuecomment-2233431475) to get the program to validate the code. He'll verify and get back to the group. 

**https://github.com/ethereum/ERCs/pull/524**
- a bunch of changes to multiple EIPs
- @SamWilsn made a comment and closed

**https://github.com/ethereum/EIPs/pull/8590**
- most editors already agreed to this was waiting on @lightclient 's approval
- confirmed & merged!

**https://github.com/ethereum/EIPs/pull/8673**
- Incorrect repo
- PR closed

**https://github.com/ethereum/ERCs/pull/540** 
- merged

**https://github.com/ethereum/EIPs/issues/8734**
- spam, closed

**https://github.com/ethereum/EIPs/pull/8724**
- Issue with auto merge, force merged.

#### Call for Input

**https://github.com/ethcatherders/EIPIP/issues/343**
- 3 editors are in favor, no one opposed. Deadline is July 28

**https://github.com/ethcatherders/EIPIP/issues/345**
- 3 editors agreed to **Delete** the EIP from the EIP GitHub repo.

**https://github.com/ethcatherders/EIPIP/issues/348**
- most of the editors are in favor, but, couldn't get any author's approval.
- The proposal is to merge this with 3 editors approval even in absence of the authors.
- @poojaranjan proposed to make it a rule that editors may force merge such PRs but, @SamWilsn wants to take it case by case basis. 

### 2. Other discussions and updates from past meetings
####  Working Group update
- Encourage and onboard `Reviewers` to support each GitHub repo
- Reviewers may leave comments to help EIP Editors
- Non-editorial PR approval tag can be created to support EIP Editors.

#### Security Considerations and Security Disclosures in EIPs ref: https://github.com/ethcatherders/EIPIP/issues/346#issuecomment-2228975727
- @Dexaran provided summary and proposals
- Detailed comment added [here](https://github.com/ethcatherders/EIPIP/issues/340#issuecomment-2207486824) and [here](https://github.com/ethcatherders/EIPIP/issues/346#issuecomment-2228975727)
- A [Call For Input](https://github.com/ethcatherders/EIPIP/issues/349) opened to collect input for decision making 
 - to allow modifications to Final proposals when all of the following are met:
    - The modification is only appending new content to the Security Considerations section;
    - The modification describes a consideration that has caused financial loss.
EIP Editors have the final say but the input here will help editors to make decision. 

### 3. EIPs Insight - Monthly EIPs status reporting.
- Check [EIPsInsight.com](https://eipsinsight.com/insight/2024/7)

## Next Meeting on Aug 21, 2024 at 14:00 UTC
- Editors agree to have once a month meeting thus, the group decided to have the meeting on every 3rd Wednesday of the month.

**Comment by poojaranjan** [comment]
Closing in favor of #350
